terraform/workspace-variables/staging_app_env.yml

```diff
@@ -10,3 +10,4 @@ DATABASE_URL: postgres://postgres@localhost:5432
 DOMAIN: s-manage-training-for-early-career-teachers.education.gov.uk
 GOVUK_WEBSITE_ROOT: s-manage-training-for-early-career-teachers.education.gov.uk
 GOVUK_APP_DOMAIN: s-manage-training-for-early-career-teachers.education.gov.uk
+SEND_EMAILS_TO: cpd-test@digital.education.gov.uk
```

Comment:
I'm not sure we want this in staging? We should check what the wider team want here.

Also, are we sure we want to introduce another difference between staging and production?

Reply:
That is a good question. I'd guess it all depends on what we use staging for. I'd imagine we'd be using it for stakeholders demo (instead of dev or PR app as we do now)?

Reply:
Right, so it seem that staging is supposed to be used for QA and PO acceptance purpose. If this is the case than we definitely want to redirect emails in staging. I don't think it introduces another difference between staging and production - we already use different Notify API key to prevent external emails from being sent so the behaviour is already different.

We might however want to send emails to different address than cpd-test, maybe cpd-staging?